config/initializers/secure_headers.rb

```diff
@@ -13,9 +13,14 @@
     :enforce     => true,
     :default_src => 'self',
     :frame_src   => 'self',
-    :connect_src => 'self ws: wss:',
+    :connect_src => %w(self ws: wss:),
     :style_src   => 'inline self',
-    :script_src  => 'eval inline self',
-    :img_src     => ['self', '*.gravatar.com']
+    :script_src  => %w(eval inline self),
+    :img_src     => %w(self *.gravatar.com)
   }
+  if Rails.env.development? #allow webpack dev server provided assets
+    dev_server = "http://#{::Rails.configuration.webpack.dev_server.host.call}:#{::Rails.configuration.webpack.dev_server.port}"
```

Comment:
0.0.0.0 is a bad default IMO, it's just not necessary to listen on all interfaces when not necessary. Sometimes I find it annoying that Puma (the server I use in dev) doesn't listen there by default but it's a saner default, so :+1: to keeping it to just localhost in dev